bootiso

```diff
@@ -413,6 +430,92 @@ assertISOIsOK() {
 	fi
 }
 
+checkISOHash() {
+	checkHash() {
+		typeset hashPath=$1 # Path to file containing hashes
+		typeset isoName=$2  # File to be checked
+		typeset hashName=$3 # Name of command of hash
+
+		# Hash from hash file
+		typeset gHash=$(awk -v pattern="$isoName$" '$0 ~ pattern { print $1; exit }' "$hashPath")
```

Comment:
I think isoName should be escaped here somehow, it could contain special regex characters. Searching for a filename with a '$' fails to find the filename in the hash file.
I'd also like to match more specific '^\w{1,}\s{2}$isoName$', but no idea how to do that with grep. Any ideas?